staging/src/k8s.io/kubectl/pkg/cmd/drain/drain.go

```diff
@@ -321,12 +319,21 @@ func (o *DrainCmdOptions) RunDrain() error {
 
 func (o *DrainCmdOptions) deleteOrEvictPodsSimple(nodeInfo *resource.Info) error {
 	list, errs := o.drainer.GetPodsForDeletion(nodeInfo.Name)
+	if len(list.Pods()) > 0 && o.drainer.DryRun {
+		fmt.Fprintf(o.Out, "namespace/pod (dry run)\n")
+		for _, pod := range list.Pods() {
+			fmt.Fprintf(o.Out, "%s/%s\n", pod.Namespace, pod.Name)
+		}
+	}
```

Comment:
This entire if should be after the below error check, in the normal when you encounter an error we'll return that error, same applies for dry-run.

Reply:
ah right, doh, thanks, updated